### DISCUSSION TITLE
Webmin authentication by unix groups

### DISCUSSION THREAD

**Discussion by atpk1989** [webmin/webmin#1899] [Ideas]
Hi all

It would be nice to have this feature of granting webmin login (and access to specific modules, etc) by unix groups, i.e., for example, all the members of unix group "network" should be allowed to login via unix authentication, with access to specific modules, without the need of creating individual webmin accounts for all users in that group.

Thanks.

**Comment by jcameron** [comment]
This is supported already - just go to Webmin -> Webmin Users -> Unix User Authentication, and you can allow members of a group to login with permissions of an underlying Webmin user.
